.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+inherit_from:
+  - https://raw.githubusercontent.com/moneyadviceservice/mas-standards/master/.rubocop.yml
+  - .rubocop_todo.yml
+require: rubocop-rspec
```

Comment:
This is interesting - I wonder if we should add this to the mas-standards yml, making it a standard part of all engine builds?

Reply:
Good point, I saw it in some projects and ended adding it everywhere I add rubocop, but is not officially a standard. I would add it unless someone disagrees with it, but I don't think is the case. Will open a PR then and check if someone has any issue.